modules/constants.py

```diff
@@ -0,0 +1,60 @@
+import importlib.resources as pkg_resources
+import json
+
+# TODO remove 'experimental' before going live
+big_query = {
```

Comment:
Nit (change throughout)
```suggestion
bigquery = {
```